docs/release-notes/v0.6.0.rst

```diff
@@ -95,7 +95,7 @@ CLI API
 
    $ pyhf --citation
    @software{pyhf,
-     author = "{Heinrich, Lukas and Feickert, Matthew and Stark, Giordon}",
+     author = {Lukas Heinrich and Matthew Feickert and Giordon Stark},
```

Comment:
Note that this won't match the output for `v0.6.0` but the reason I'm suggesting to change it here is to avoid copy-paste people from finding the wrong version. Thoughts?

Reply:
seems fine.